persistence-jdbc/src/main/scala/zio/actors/persistence/jdbc/JDBCJournal.scala

```diff
@@ -0,0 +1,77 @@
+package zio.actors.persistence.jdbc
+
+import cats.effect.Blocker
+import com.zaxxer.hikari.HikariDataSource
+import doobie._
+import doobie.hikari.HikariTransactor
+import doobie.implicits._
+
+import zio.{ DefaultRuntime, IO, Promise, Task, ZIO }
+import zio.actors.ActorSystemUtils
+import zio.actors.persistence.PersistenceId.PersistenceId
+import zio.actors.persistence.journal.Journal
+import zio.actors.persistence.jdbc.JDBCConfig.DbConfig
+import zio.blocking.Blocking
+import zio.interop.catz._
+
+private[actors] object JDBCJournal {
+
+  private lazy val runtime           = new DefaultRuntime {}
+  private lazy val transactorPromise = runtime.unsafeRun(Promise.make[Exception, HikariTransactor[Task]])
+
+  def makeTransactor(dbConfig: DbConfig): ZIO[Blocking, Throwable, HikariTransactor[Task]] =
```

Comment:
This is very "leaky" interface. Preferably we'll expose only a single factory method that returns an instance of a journal.

Reply:
Sure, I will make those methods private and only expose Journal constructor